test/dlc_tests/eks/pytorch/inference/test_eks_pytorch_inference.py

```diff
@@ -2,18 +2,71 @@
 import random
 
 import pytest
+from time import sleep
 
 from invoke import run
 
 import test.test_utils.eks as eks_utils
 import test.test_utils as test_utils
 
+@pytest.mark.model("resnet")
+def test_eks_pytorch_neuron_inference(pytorch_inference, neuron_only):
+    server_type = test_utils.get_inference_server_type(pytorch_inference)
+    if "eia" in pytorch_inference or "neuron" not in pytorch_inference:
+        pytest.skip("Skipping EKS Neuron Test for EIA and Non Neuron Images")
```

Comment:
Since the Neuron test isn't applicable to EI images, why not just make this if-condition conditioned on `"neuron" not in pytorch_inference`?

Also, please change the message printed out to say "EKS Neuron Inference test is not applicable to EI images" or something similar.

Reply:
Done.